helm/cluster-operator/templates/deployment.yaml

```diff
@@ -4,21 +4,18 @@ metadata:
   name: {{ tpl .Values.resource.default.name  . }}
   namespace: {{ tpl .Values.resource.default.namespace  . }}
   labels:
-    app: {{ .Values.project.name }}
-    version: {{ .Values.project.version }}
+    {{- include "cluster-operator.labels" . | nindent 4 }}
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: {{ .Values.project.name }}
-      version: {{ .Values.project.version }}
+      {{- include "cluster-operator.selectorLabels" . | nindent 6 }}
   strategy:
-    type: Recreate
+    type: RollingUpdate
```

Comment:
Nope. We want `Recreate`. Why did that change?

Reply:
It was changed in giantswarm/aws-operator/pull/2171 before. Reverting.